src/logic/reminders.ts

```diff
@@ -0,0 +1,82 @@
+import { isTest } from '../utils/env';
+
+const CREATE_ONE_OFF_REMINDER_ENDPOINT = 'https://support.theguardian.com/reminders/create/one-off';
```

Comment:
Should this vary based on the environment? I guess there's a CODE equivalent? For other similar API endpoints, we thread these through from the platform (which has the stage awareness), for example the subscribe to newsletter endpoint.

Reply:
Agreed! Will look into it

Reply:
A fix is in place - we now get the environment as an attribute from DCR